slinky-utils/src/main/scala/net/wiringbits/webapp/utils/slinkyUtils/facades/reactrouterdom/Route.scala

```diff
@@ -0,0 +1,25 @@
+package net.wiringbits.webapp.utils.slinkyUtils.facades.reactrouterdom
+
+import slinky.core.ExternalComponent
+import slinky.core.facade.ReactElement
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+import scala.scalajs.js.|
+
+@JSImport("react-router-dom", "Route")
+@js.native
+private object RouteComponent extends js.Object
+
+object Route extends ExternalComponent {
+  case class Props(
+      path: String,
+      exact: Boolean,
+      children: ReactElement
+  )
+  override val component: String | js.Object = RouteComponent
+
+  def apply(path: String, exact: Boolean = false, children: ReactElement): ReactElement = {
```

Comment:
This should likely be `def apply(path: String, exact: Boolean = false)(children: => ReactElement)`